python/pylibcudf/pylibcudf/io/types.pyx

```diff
@@ -29,7 +29,7 @@ from pylibcudf.libcudf.io.types cimport (
 from pylibcudf.libcudf.types cimport size_type
 from pylibcudf.libcudf.utilities.span cimport host_span, device_span
 
-from pylibcudf.utils cimport _get_stream, _get_memory_resource
+# Utils imports removed as parameters are now required
```

Comment:
Is this comment needed? Maybe extraneously added by AI?

Reply:
Ugh I told AI to remove these comments after I noticed them the first time. I guess they snuck back in. Thanks!